package.json

```diff
@@ -4,12 +4,19 @@
   "main": "chronobot.js",
   "scripts": {
     "build": "tsc",
-    "start": "node chronobot.js"
+    "start": "node dist/chronobot.js",
+    "login": "node dist/login.js",
```

Comment:
So cool you did that, let's add it to the readme I guess?

Reply:
Can do, I just did very basic changes for now, I plan to add i18n possibility too since it can come handful for users that live in certain areas that speak English natively (like my wife for example XD ).